core/string/ustring.cpp

```diff
@@ -4144,6 +4144,92 @@ String String::replace_first(const char *p_key, const char *p_with) const {
 	return *this;
 }
 
+String String::replace_char(char32_t p_key, char32_t p_with) const {
+	ERR_FAIL_COND_V_MSG(p_with == 0, *this, "`with` must not be null.");
```

Comment:
I had missed this earlier; 'null' may be confusing because it usually implies a "null pointer". The common way to call the character in ustring.cpp right now seems to be `NUL`, or rather `NUL character`.

Reply:
Will do